pkg/rtc/mediatracksubscriptions.go

```diff
@@ -161,14 +162,17 @@ func (t *MediaTrackSubscriptions) AddSubscriber(sub types.LocalParticipant, wr *
 	})
 
 	// Bind callback can happen from replaceTrack, so set it up early
+	var reusingTransceiver atomic.Bool
```

Comment:
putting this back. Had removed it in previous PR for re-use of down track state cache. But, keeping the forwarder state for migration out case separate.